docs/getting_started/logging.rst

```diff
@@ -15,6 +15,9 @@ Add the following to your code to log everything available:
    logger = logging.getLogger('prawcore')
    logger.setLevel(logging.DEBUG)
    logger.addHandler(handler)
+   logger2 = logging.getLogger('praw')
```

Comment:
Consider looping over the loggers instead of dealing with `logger` and `logger2`. Also maybe we want to add a formatter now so we can easily distinguish from praw and prawcore output.